xapian-core/cluster/euclidian_sim.cc

```diff
@@ -40,86 +40,10 @@ EuclidianDistance::get_description() {
     return "Euclidian Distance metric";
 }
 
+/** TODO
+ *  Implement the euclidian distance metric for the current API
+ */
```

Comment:
I did plan to implement the euclidian distance in the upcoming commits, so just kept this as a dummy for now. I think having two distance measures can help us in some ways.